modin/engines/base/io/text/text_file_dispatcher.py

```diff
@@ -45,6 +73,24 @@ def get_path_or_buffer(cls, filepath_or_buffer):
 
     @classmethod
     def build_partition(cls, partition_ids, row_lengths, column_widths):
+        """
+        Build array with partitions of `cls.frame_partition_cls` class.
+
+        Parameters
+        ----------
+        partition_ids : list
+                array with references to the partitions data.
+        row_lengths : list
+                Partitions rows lengths.
+        column_widths : list
+                Number of columns in each partition.
+
+        Returns
+        -------
+        numpy array
```

Comment:
```suggestion
        np.ndarray
```

Reply:
This suggestion conflicts with @vnlitvinov [suggestion](https://github.com/modin-project/modin/pull/2952#discussion_r609527344). So let's define the common decision, my proposal is to make `numpy.ndarray`. What do you think?

Reply:
We should use either `NumPy array` in the description of a return value or `np.ndarray` in the description of a return value type because we import `numpy` as `np` at the top of a file. Also, plenty of people know it and use `import numpy as np` as well.

Reply:
For me it's fine to use both of these options.
@vnlitvinov are you agree to use `np.ndarray` instead of `numpy array`?

Reply:
I'm fine with any of the two as long as it's consistent and docs are generated correctly.